.github/workflows/docs.yml

```diff
@@ -0,0 +1,187 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "docs/**"
+      - "solidity/**"
+      - "!solidity/dashboard/**"
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  docs-detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      path-filter: ${{ steps.filter.outputs.path-filter }}
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event_name == 'pull_request' 
+      - uses: dorny/paths-filter@v2
+        if: github.event_name == 'pull_request' 
+        id: filter
+        with:
+          filters: | # TODO: Remove '|(.github/workflows/docs.yml)' once workflow gets tested
```

Comment:
Let's remember about this TODO.

Reply:
Removed, see 0d01d5f5d9b81843e5c1cdc9b0fe1daa58a502bd.